lerna-package-lock.json

```diff
@@ -818,9 +818,9 @@
 			},
 			"dependencies": {
 				"@fluidframework/protocol-definitions": {
-					"version": "0.1012.1",
-					"resolved": "https://registry.npmjs.org/@fluidframework/protocol-definitions/-/protocol-definitions-0.1012.1.tgz",
-					"integrity": "sha512-L5BVD9YYvossY0TfSMyT4j7VvbOmD7lvmJVNcqitNTKnPYtfjuK6X7t/o22Yc4llxhlorysqeVUrd23T/Iy4XA==",
+					"version": "0.1012.2",
```

Comment:
This doesn't seem bad but I don't really know why these are bumping to 0.1012.2, or why they're currently still at 0.1012.1 when it looks like all our packages already reference ^0.1013.0-0.  @curtisman is this normal?